src/Sulu/Bundle/SnippetBundle/Snippet/SnippetRepository.php

```diff
@@ -77,7 +77,7 @@ public function getReferences($uuid)
      *
      * UUIDs which fail to resolve to a snippet will be ignored.
      *
-     * @param string $locale
+     * @param ?string $locale
```

Comment:
the locale is required here. so would not add any `null` here.

Reply:
still open, we should keep this also when currently the locale is not handled on all cases correctly:

```suggestion
     * @param string $locale
```